### PR TITLE
remove default headers

### PR DESCRIPTION
**Pull request by robinphillips**
Removing the default headers from new files in Xcode.

Removing remaining existing headers.

Replacing launch screen storyboard with a Launch Screen plist.

### CODE REVIEW

File: Homepage-DemoApp/Homepage-DemoApp/LaunchScreen.swift
Comment (batesyuk) [inline Homepage-DemoApp/Homepage-DemoApp/LaunchScreen.swift:1]: Can you get away without having the LaunchScreen storyboard? I thought it was a requirement for the splash screen to be in a storyboard?

Comment (robinphillips) [reply Homepage-DemoApp/Homepage-DemoApp/LaunchScreen.swift:1]: Yeah, I remembered there was a way of removing the storyboards but forgot it was weird. It's a plist which I think is probably fine for these package repos?

Comment (batesyuk) [reply Homepage-DemoApp/Homepage-DemoApp/LaunchScreen.swift:1]: I'm not sure it's working. Just goes straight to the Homepage.

Comment (robinphillips) [reply Homepage-DemoApp/Homepage-DemoApp/LaunchScreen.swift:1]: It is for me 🤔
https://github.com/alphagov/govuk-mobile-ios-homepage/assets/104073334/d27a1b65-631d-4817-8d99-c8507d8583a2

**Review by batesyuk — APPROVED** [review]

**Pull request merged**